### ISSUE TITLE
[BUG] TARDIS Dimension always set to raining

### ISSUE THREAD

**Issue by chaNcharge** [DarkholmeTenk/TardisModIssues#531]
This isn't exactly noticable because rain is disabled in the TARDIS, but if you have NEI installed, you'll notice it. If you step in the TARDIS with NEI installed, it'll show that rain is enabled. This can cause problems with mods that add in special rain sounds, such as MAtmos. This is in the latest dev version of the mod, build 352

To reproduce:
1. Install Tardis mod and NEI
2. Create a new world
3. Spawn in a TARDIS with TARDIS key
4. Step inside
5. You'll see that NEI and Minecraft detects rain is on, but that's not all
6. Step back outside
7. Minecraft thinks rain is on in the world, not visibly, but can trigger special rain sounds from MAtmos

The issue is resolved temporarily by relogging or turning rain on and off

This issue is the exact same as #515 but it appears to have not been fixed since then.

**Comment by DarkholmeTenk** [comment]
I'm sorry, but this makes no sense. It's clearly not raining. Minecraft does not seem to think it is raining, in any way.

I set the rain strength to 0, I set the time to next rain to Integer.MAX_VALUE. I do this for every updateWeather call. Calling isRaining() on the world returns false.

Clicking the weather button in NEI says "Rain turned on", suggesting NEI knows the world is not raining.

The bug appears to be something weird with the integrated server when teleporting.

**Comment by chaNcharge** [comment]
Yes, I see. I'm going to try and post my log files, see if there's anything wrong with it. It does say "Rain turned on" but notice that the button is already highlighted saying rain is already on. Try installing MAtmos along with the resource pack that is required for MAtmos to work and place down some leaves. It may be something to do with NEI, I'm going to keep testing this, but for now, here's the log files.
latest.log: https://drive.google.com/file/d/0B3ZoX-8LP704MV9Yb0xBNWFtTDA/view?usp=sharing
fml-client-latest.log: https://drive.google.com/file/d/0B3ZoX-8LP704VzFWWC1TWXJiQUk/view?usp=sharing
Launcher Log: https://drive.google.com/file/d/0B3ZoX-8LP704WTQ2ODRzN0VER2s/view?usp=sharing

**Comment by chaNcharge** [comment]
Okay, so I've done some testing, I've taken out NEI, added in MAtmos to try and tell if it is raining, turns out it is not NEI, but it's somewhere within the mod or minecraft itself. As a temporary solution, I've taken out rain on leaves sounds so it's not as bothersome.
